packages/flutter_test/lib/src/test_pointer.dart

```diff
@@ -169,6 +169,25 @@ class TestPointer {
       delta: delta,
     );
   }
+
+  /// Create a [PointerScrollEvent] with the given delta.
+  ///
+  /// By default, the time stamp on the event is [Duration.zero]. You can give a
+  /// specific time stamp by passing the `timeStamp` argument.
+  PointerScrollEvent scroll(
```

Comment:
the docs here should say something about how this refers to "mouse wheel"-style scrolling, not finger-drag-scrolling.

Reply:
Done.